app/controllers/invitations_controller.rb

```diff
@@ -57,15 +57,13 @@ def reject
   def rsvp_meeting
     return redirect_to :back, notice: 'Please login first' unless logged_in?
 
-    meeting = Meeting.find_by(slug: params[:meeting_id])
+    invitation = load_invitation
+    meeting = invitation.meeting
 
-    invitation = MeetingInvitation.find_or_create_by(meeting: meeting, member: current_user, role: 'Participant')
-
-    invitation.update_attribute(:attending, true)
-
-    if invitation.save
+    if invitation.update(attending: true)
```

Comment:
```suggestion
    invitation.assign_attributes(attending: true)
    
    if invitation.save
```
As `update` returns the resulting object back whether that is updated or not, the `else` clause of this statement will never be reached

Reply:
> The resulting object is returned whether the object was saved successfully to the database or not. ([source](https://apidock.com/rails/ActiveRecord/Base/update/class))

You are right, I was certain it would return `true/false`… I’ll revert this particular change now. 👍 

P.S.: Thank you so much for the code reviews! ⚡

Reply:
Actually, @despo, are you sure? I was double-checking on my console and I seem to get a `true/false` instead:
```rb
# successful update
irb(main):009:0> Member.last.update(surname: "Morais")
D, [2020-12-04T09:37:14.202862 #96295] DEBUG -- :   Member Load (1.2ms)  SELECT  "members".* FROM "members"  ORDER BY "members"."id" DESC LIMIT 1
D, [2020-12-04T09:37:14.204774 #96295] DEBUG -- :    (0.4ms)  BEGIN
D, [2020-12-04T09:37:14.211939 #96295] DEBUG -- :   AuthService Load (0.8ms)  SELECT "auth_services".* FROM "auth_services" WHERE "auth_services"."member_id" = $1  [["member_id", 574]]
D, [2020-12-04T09:37:14.216121 #96295] DEBUG -- :   Member Exists (0.7ms)  SELECT  1 AS one FROM "members" WHERE ("members"."email" = 'gnclmorais@gmail.com' AND "members"."id" != 574) LIMIT 1
D, [2020-12-04T09:37:14.222830 #96295] DEBUG -- :   SQL (0.5ms)  UPDATE "members" SET "surname" = $1, "updated_at" = $2 WHERE "members"."id" = $3  [["surname", "Morais"], ["updated_at", "2020-12-04 09:37:14.217152"], ["id", 574]]
D, [2020-12-04T09:37:14.224083 #96295] DEBUG -- :   ActsAsTaggableOn::Tagging Load (0.4ms)  SELECT "taggings".* FROM "taggings" WHERE "taggings"."taggable_id" = $1 AND "taggings"."taggable_type" = $2  [["taggable_id", 574], ["taggable_type", "Member"]]
D, [2020-12-04T09:37:14.225752 #96295] DEBUG -- :    (0.9ms)  COMMIT
=> true

# unsuccessful update
irb(main):010:0> Member.last.update(email: nil)
D, [2020-12-04T09:38:09.206675 #96295] DEBUG -- :   Member Load (1.4ms)  SELECT  "members".* FROM "members"  ORDER BY "members"."id" DESC LIMIT 1
D, [2020-12-04T09:38:09.208022 #96295] DEBUG -- :    (0.3ms)  BEGIN
D, [2020-12-04T09:38:09.211060 #96295] DEBUG -- :   AuthService Load (0.5ms)  SELECT "auth_services".* FROM "auth_services" WHERE "auth_services"."member_id" = $1  [["member_id", 574]]
D, [2020-12-04T09:38:09.216430 #96295] DEBUG -- :   Member Exists (0.6ms)  SELECT  1 AS one FROM "members" WHERE ("members"."email" IS NULL AND "members"."id" != 574) LIMIT 1
D, [2020-12-04T09:38:09.217502 #96295] DEBUG -- :    (0.4ms)  ROLLBACK
=> false
```

Reply:
@gnclmorais haha, it seems like I'm a couple of years out of date on the documentation of update. It does indeed do that now, I was just used to working around it and google bringing back the old version result first didn't help!